File: Java/Lista_03(Estrutura_Repetitiva)/Tabuada.java
import java.util.Scanner;

public class Tabuada {

	public static void main(String[] args) {

		Scanner s = new Scanner(System.in);
		
		System.out.print("Deseja a tabuada para qual valor? ");
		int n = s.nextInt();
		
		for(int i=1;i<=10;i++) {
			System.out.println(n + " x " + i + " = " + (n*i));
		}
		
		s.close();
		
	}

}

File: C/Lista_03(Estrutura_Repetitiva)/Tabuada.c
#include <stdio.h>

int main () {

    int n;

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d", &n);

    for(int i=1;i<=10;i++){
        printf("%d x %d = %d\n", n, i, (n*i));
    }

    return 0;

}